python_scripts/cross_validation_baseline.py

```diff
@@ -75,6 +75,47 @@
 )
 errors_dummy_regressor.describe()
 
+# %% [markdown]
+# Something interesting happens if we measure the generalization performance
+# using R² rather than mean absolute error:
+
+# %%
+result_dummy = cross_validate(
+    dummy, data, target, cv=cv, scoring="r2", return_train_score=True, n_jobs=2
+)
+r2_train_score_dummy_regressor = pd.Series(
+    result_dummy["train_score"], name="Dummy regressor"
+)
+r2_train_score_dummy_regressor.describe()
+
+# %% [markdown]
+# The R² score is always 0. It can be shown that this is always the case,
+# because of the [mathematical definition of the R²
+# score](https://scikit-learn.org/stable/modules/model_evaluation.html#r2-score-the-coefficient-of-determination).
+# This helps put your model's R² score in perspective: if your model has
+# an R² score higher than 0 then it performs better
+# than a `DummyRegressor` with `strategy="mean"`; similarly, if the
+# R² score is lower than 0 then your model is worse than the dummy
+# regressor.
+
+# %% [markdown]
+# ```{admonition} Mathematical explanation
+# :class: dropdown
+# Recall that the R² score is defined as
+# $$
```

Comment:
It does not render as expected. It looks like it only consider a single `$`

Reply:
I don't know if it can support multiline equations. Here is the rendering from the CI preview:

![image](https://github.com/user-attachments/assets/915a8a34-6405-4866-b671-f790f75fa78d)

https://pull-request-819--scikit-learn-mooc.netlify.app/python_scripts/cross_validation_baseline

Reply:
The documentation says it should work:

https://jupyterbook.org/en/stable/content/math.html#math-blocks

Reply:
Maybe it's a bad interaction with the use of the admonition?

Reply:
I went to look at the CI preview (link above) and it looks like this:

![2025-04-03T10_41_33_screenshot](https://github.com/user-attachments/assets/0741500e-b630-4c8b-b44a-7d0bc9e59389)

Reply:
Now it looks like there's a weird vertical space before the first equation

Reply:
So removing the double dollar appears to be fine.

Interestingly, on firefox, here is what I get:

![image](https://github.com/user-attachments/assets/26d4eebb-1738-4f27-8061-79bf53feee69)

I think this is good enough either way.